apps/io-wallet-user-func/local.settings.json.example

```diff
@@ -29,6 +29,7 @@
     "EntityConfigurationStorageAccount__serviceUri": "ENTITY_CONFIGURATION_STORAGE_ACCOUNT__SERVICE_URI",
     "EntityConfigurationStorageContainerName": "ENTITY_CONFIGURATION_STORAGE_CONTAINER_NAME",
     "StorageConnectionString": "STORAGE_ACCOUNT_CONNECTION_STRING",
+    "SendEmailQueueName": "WALLET_INSTANCE_ACTIVATION_EMAIL_QUEUE",
```

Comment:
I know we previously said something else, but since we will also send the email for revocation, maybe we should name the queue something like `WalletInstanceCreationQueueName`